docs/source/release/v5.1.0/direct_geometry.rst

```diff
@@ -30,5 +30,6 @@ Bugfixes
 --------
 
 - A bug introduced in v5.0 causing error values to tend to zero on multiple instances of :ref:`Rebin2D <algm-Rebin2D>` on the same workspace has been fixed.
+- A bug in the validation of temporary workpaces for :ref:`MDNorm <algm-MDNorm>` has been fixed. One can now use temporary workspaces when slicing in any order.
```

Comment:
I just want to confirm that this release note will make sense to any user that is checking it. because It is not entirly clear to me.

Reply:
This will allow slicing with dimension0 `DeltaE` and use temporary workspaces. Before `DeltaE` had to be dimension3 when using temporary workspaces, otherwise validation will fail. I am willing to change the release notes if a better option is suggested.